libs/sync/src/lib/sync-details-page/sync-details-page.component.ts

```diff
@@ -97,10 +98,15 @@ export class SyncDetailsPageComponent implements OnDestroy {
   }
 
   triggerSync() {
-    this.syncId
-      .pipe(
-        take(1),
-        switchMap((syncId) => this.syncService.triggerSync(syncId)),
+    this.modalService
+      .openModal(SyncModalComponent)
```

Comment:
I'm wondering why you didn't just used a `confirmDialog` here? the configuration model allows to customize both footer buttons.